### DISCUSSION TITLE
`pac connector init` only generates `apiProperties.json`

### DISCUSSION THREAD

**Discussion by Laskewitz** [microsoft/powerplatform-vscode#325] [Preview]
I tried out the `pac connector` commands. I really like them. The biggest pro for me is that I don't have to auth every xx minutes anymore. 😇

I have some feedback about the `pac connector init` command: the command only generates `apiProperties.json` file, but I would expect to have a `apiDefinition.swagger.json` file in there as well. Is that something that can be added?

Maybe this could be the contents of the file:

```
{
  "swagger": "2.0",
  "info": {
    "title": "Connector Name",
    "description": "Description.",
    "version": "1.0",
    "contact": {
      "name": "John Doe",
      "url": "https://company.com/",
      "email": "email@company.com"
    }
  },
  "x-ms-connector-metadata": [
    {
      "propertyName": "Website",
      "propertyValue": "https://your-website-here.com/"
    },
    {
      "propertyName": "Privacy policy",
      "propertyValue": "https://privacy-policy-link-here.com/"
    },
    {
      "propertyName": "Categories",
      "propertyValue": "Category1;Category2"
    }
  ],
  "host": "url.host.com",
  "basePath": "/",
  "schemes": [
    "https"
  ],
  "consumes": [],
  "produces": [],
  "paths": {},
  "definitions": {},
  "responses": {},
  "securityDefinitions": {},
  "security": [],
  "tags": []
}
```

**Comment by filcole** [comment]
I wonder if the expectation is for the Swagger/OpenAPI definitions to be created via a automatically via a script/CLI ideally within a pipeline, e.g. see this PR for Azure Functions OpenAPI extensions: https://github.com/Azure/azure-functions-openapi-extension/pull/489 now in release.

**Comment by Laskewitz** [comment]
Would be nice to have an option for both.

**Comment by davidjenni** [comment]
@Laskewitz @filcole note that the `connector` noun and its verbs are still in early preview, with work still ongoing both on the Dataverse side and in the pac CLI.

I'm moving this thread into the discussion area

CC: @tehcrashxor @kkanakas
